packages/communication-react/CHANGELOG.stable.md

```diff
@@ -7,104 +7,45 @@ This log was last generated on Thu, 14 Mar 2024 03:50:48 GMT and should not be m
-- Fixed error when spoken language and caption language show up empty on initiation  ([PR #4228](https://github.com/azure/communication-ui-library/pull/4228) by 96077406+carocao-msft@users.noreply.github.com)
+- Fix missing alternate Caller id ([PR #4216](https://github.com/azure/communication-ui-library/pull/4216) by dmceachern@microsoft.com)
+- Resolve issue where edited label is missing from received messages ([PR #4150](https://github.com/azure/communication-ui-library/pull/4150) by 73612854+palatter@users.noreply.github.com)
+- Fix error when spoken language and caption language show up empty on initiation  ([PR #4228](https://github.com/azure/communication-ui-library/pull/4228) by 96077406+carocao-msft@users.noreply.github.com)
 - Update Calling sounds to not play end call sound when transfer happens ([PR #4163](https://github.com/azure/communication-ui-library/pull/4163) by dmceachern@microsoft.com)
-- Resolve issue where images previews would not sure in a message if the message also had a file attachment. ([PR #4167](https://github.com/azure/communication-ui-library/pull/4167) by 73612854+palatter@users.noreply.github.com)
-- Bug Fix metadata was not being sent up to the handler ([PR #4213](https://github.com/azure/communication-ui-library/pull/4213) by 9044372+JoshuaLai@users.noreply.github.com)
-- Fixed the issue where image placeholder being shown when fetching failed ([PR #4172](https://github.com/azure/communication-ui-library/pull/4172) by 109105353+jpeng-ms@users.noreply.github.com)
-- Add key to MsftMention tag ([PR #4188](https://github.com/azure/communication-ui-library/pull/4188) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-- BugFix on showing broken img icon into the image gallery on error ([PR #4249](https://github.com/azure/communication-ui-library/pull/4249) by 9044372+JoshuaLai@users.noreply.github.com)
-- Resolve issue where envHelper would return early on error throw when appsettings.json doesn't exist ([PR #4161](https://github.com/azure/communication-ui-library/pull/4161) by 73612854+palatter@users.noreply.github.com)
-- Revert "[Chat] Wrap ImageOverlay with theme provider and change some of the APIs (#4117)" ([PR #4131](https://github.com/azure/communication-ui-library/pull/4131) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
 - Fix hanging express.close if have unclosed connection ([PR #4138](https://github.com/azure/communication-ui-library/pull/4138) by 77021369+jimchou-dev@users.noreply.github.com)
-- Remove CC in Storybook ([PR #4200](https://github.com/azure/communication-ui-library/pull/4200) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-- Fix crush when inline image is deleted from another client ([PR #4261](https://github.com/azure/communication-ui-library/pull/4261) by 107075081+Leah-Xia-Microsoft@users.noreply.github.com)
-
 
 ## [1.13.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.13.0)
```

Comment:
Might as well fix this link to be https://github.com/azure/communication-ui-library/tree/1.13.0 😅

Reply:
done